genie-core/src/main/java/com/netflix/genie/core/jpa/services/JpaClusterServiceImpl.java

```diff
@@ -318,6 +318,68 @@ public void removeAllConfigsForCluster(
      * {@inheritDoc}
      */
     @Override
+    public void addDependenciesForCluster(
```

Comment:
Not saying you need to do this as part of this PR but we have so many of these repeatable crud methods it would be nice to find a way to share implementations across clusters, commands, applications...